ceasuri.css

```diff
@@ -0,0 +1,92 @@
+
+.container {
+    width: 500px;
+    height: 250px;
+    border: 1px solid rgb(104, 98, 98);
+    padding: 5px;
+    display: flex;
+    margin: 10px;
+    justify-content:space-between;
+    align-items: end;
+    position:relative;
+    top: 15%;
+    left: 15%;
+    background-color: white;
+}
+
+.container div {
+    border: 1px solid rgb(75, 71, 71);
+    font-size: 20px;
+    padding: 10px;
+    height: 150px;
+    width: 100px;
+    box-shadow: 10px;
+}
+
+
+#box1{
+   background-image: url(https://storage0.dms.mpinteractiv.ro/media/1/1481/10368/18675259/9/maitres-du-temps-chapter-one-round-transparence.jpg?height=327&width=620);
```

Comment:
pentru tema nu e o abordare gresita, insa cand vei folosii baze de date, imaginilie vor fi stocate pe un server si o sa fie greu sa le setezi in css.

o abordare mai corecta ar fi fost sa folosesti tagul <img src="link_imagine" />